utils/sl/prereq_tools/base.py

```diff
@@ -819,9 +819,14 @@ def has_source(self, env, *comps, **kw):
     def _setup_build_type(self):
         """set build type"""
         self.add_opts(EnumVariable('BUILD_TYPE', "Set the build type",
-                                   'dev', ['dev', 'debug', 'release'],
+                                   'dev', ['dev', 'debug', 'release', 'asan'],
```

Comment:
Is there a reason this wouldn't go in the debug or dev builds?

I wonder if we shouldn't have a list of build options, and have dev,debug,release map to a set of options.

Reply:
Not a bad idea to have a map